cypress/integration/Modal.spec.ts

```diff
@@ -156,6 +156,17 @@ describe('Modal', () => {
             cy.findByRole('dialog', {name: 'MIT License'}).should('not.exist');
           });
         });
+
+        context(`when clicking outside the modal on mobile view`, () => {
+          beforeEach(() => {
+            cy.viewport(380, 720);
```

Comment:
Is this supposed to be `480, 720` ?

Reply:
It does not really matter, I just need to simulate any phone screen. I changed it to specific phone screen to confuse less why I used this numbers.